src/assets/image.js

```diff
@@ -0,0 +1,17 @@
+import Logo from '../../public/logo.png';
```

Comment:
Lets the short the imports by length i ascending order . You can use https://marketplace.visualstudio.com/items?itemName=pratishshr.bharyang-vscode .

Reply:
🆗